src/main/kotlin/com/evalify/evalifybackend/questions/domain/Coding.kt

```
@@ -0,0 +1,4 @@
+package com.evalify.evalifybackend.questions.domain
+
+class Coding {
+}
```

detekt (1.23.8)

[warning] The class or object Descriptive is empty.

(detekt.empty-blocks.EmptyClassBlock)